### DISCUSSION TITLE
TOML Table Support

### DISCUSSION THREAD

**Discussion by DrDonk** [mitsuhiko/minijinja#411] [Q&A] [answered]
I've been using a single TOML file with tables to store multiple boot configurations. For (simplified) example:

```
[default]
debug = 'false'

[boot-debug]
debug = 'true'
```

Is there a way I can specify which table to use in minijinja-cli? Currently I am using jinja2-cli https://github.com/mattrobenolt/jinja2-cli which has a --section parameter to allow a specific TOML table to be used. The option also works on other supported types such as YAML.

I would like to remove the dependency on Python and minijinja-cli does everything I need apart from this one feature. I could split the TOML files but would prefer to keep the settings in one file. 

Any thoughts?

**Comment by mitsuhiko** [comment, marked as answer]
That's a good idea. I added this: https://github.com/mitsuhiko/minijinja/pull/421
